README.md

```diff
@@ -78,7 +82,7 @@ To read about using Starter Workflows, see [here](https://docs.github.com/en/act
 
 1. Publish
     - optional arguments:
-        - `node-version`: The version of Node the package is to be published with. This is defaulted to the latest version of NodeJS 16.
+        - `node_version`: The version of Node the package is to be published with. This is defaulted to the latest version of NodeJS 18.
```

Comment:
note @JoshuaLicense  comment above

Reply:
Team decided during meeting to use camel-case throughout.